include/openmc/particle_data.h

```diff
@@ -364,7 +363,7 @@ class ParticleData {
   const double& time() const { return time_; }
   double& time_last() { return time_last_; }
   const double& time_last() const { return time_last_; }
-  bool& alive() { return alive_; }
+  bool alive() { return wgt_ > 0.0; }
```

Comment:
I think it should be wgt_ != 0, since a negative weight particle is not impossible for some pathological cases in multigroup with Legendre angular distributions, right?

Reply:
Fair point; I'll make that change.